packages/chatbot-server-mongodb-public/evalCases/uni_skills_evaluation_questions.yml

```diff
@@ -0,0 +1,188 @@
+  reference: There is a range of skills available on our skills page. We are
+    building these out starting with several on data modeling topics. The skill
+    in data modeling we'd recommend you start with is "From Relational Model
+    (SQL) to MongoDB's Document Model". The other data modeling skills cover the
+    topics of Schema Patterns and Antipatterns, Advanced Schema Patterns and
+    Antipatterns, and Schema Design Optimization.
+  expectedLinks:
+    - https://learn.mongodb.com/skills
+- name: What skill badge is available to earn on the document model?
+  messages:
+    - role: user
+      content: What skill badge is available to earn on the document model?
+  reference: The skill, "From Relational Model (SQL) to MongoDB's Document Model",
+    teaches you about the document model. It covers how to model your workload,
+    how to design data relationships, and how to validate your schemas. You will
+    learn about the MongoDB metholodgy for converting from a relational model to
+    a document model.
+  expectedLinks:
+    - https://learn.mongodb.com/skills
+    - https://learn.mongodb.com/courses/relational-to-document-model
+- name: What skill badge is available to earn on schema patterns?
+  messages:
+    - role: user
+      content: What skill badge is available to earn on schema patterns?
+  reference: The skill, "MongoDB Schema Design Patterns and Antipatterns", teaches
+    you about the schema patterns for the document model. It covers the three
+    most common schema design patterns, specifically the inheritance pattern,
+    the computed pattern, and the extended reference pattern. It teachs you when
+    and how you should apply them in your data model.
+  expectedLinks:
+    - https://learn.mongodb.com/skills
+    - https://www.mongodb.com/docs/manual/data-modeling/design-patterns/
+    - https://learn.mongodb.com/courses/schema-design-patterns-and-antipatterns
+    - https://learn.mongodb.com/courses/advanced-schema-patterns-and-antipatterns
+    - https://learn.mongodb.com/courses/schema-design-optimization
+- name: What skill badge do you have on the schema antipatterns?
+  messages:
+    - role: user
+      content: What skill badge do you have on the schema antipatterns?
+  reference: >-
+    The skill, "MongoDB Schema Design Patterns and Antipatterns", teaches you
+    about the antipatterns that you might find when modeling your data for the
+    document model. This includes antipatterns such as unbounded arrays and
+    bloated documents. You will learn how to identify and avoid them in your
+    schema design. 
+
+    The skill, "MongoDB Advanced Schema Design Patterns and Antipatterns", also teaches you about more complex antipatterns such as massive number of collections, unnecessary indexes, data normalization, and case sensitivity. This category of antipattern is classified as an advanced as in order to identify their performance impacts you must do more than analyse the schema.
```

Comment:
```suggestion
    The "MongoDB Schema Design Patterns and Antipatterns" skill teaches you
    about patterns to avoid when modeling your data for the document model.
    You will learn how to identify and avoid antipatterns such as unbounded
    arrays and bloated documents in your schema design. 

    The "MongoDB Advanced Schema Design Patterns and Antipatterns" skill teaches
    you about more complex antipatterns such as massive number of collections,
    unnecessary indexes, data normalization, and case sensitivity. This category of
    antipattern is classified as "advanced" because you must do more than analyze
    your schema to identify their performance impacts.
```